openstudiocore/src/model/Space.cpp

```diff
@@ -2496,6 +2514,15 @@ namespace detail {
     return result;
   }
 
+  boost::optional<ModelObject> Space_Impl::buildingUnitAsModelObject() const {
```

Comment:
these methods were to support Q_PROPERTY access, we don't support those anymore so there is no need to implement this method